src/components/Form/Crp.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <div class="d-flex flex-items-center px-4 px-md-0 mb-3">
-      <h4 class="flex-auto" v-text="'Token symbol'" />
+      <h4 class="flex-auto">{{ $t('tokenSymbol') }}</h4>
```

Comment:
Any reason you use mustache style syntax instead of `v-text`? I tend to use `v-text` when it's possible, its often shorter and easier to read.
```suggestion
      <h4 v-text="$t('tokenSymbol')" class="flex-auto"/>
```

Reply:
No reason - I will change it (I read your comment on Discord to that effect, but had already committed this).